### PR TITLE
DOCS-355: Update quickstart for Embedded Console

### LINKED ISSUE #355: Embedded Console Release
# Summary

Next stable will include embedded MinIO Console. 

# Goals

- Update installation / deployment pages to remove instructions on installing Console manually
- Revise to use embedded console behavior.

### PR DESCRIPTION
**Pull request by ravindk89**
# Summary

Fixes #355 by adding high level information on using the MinIO Console

# Goals

Remove references to manual Console deployment

# Non-Goals

- Update documentation consistently to use Console features
- Create dedicated Console documentation

### CODE REVIEW

**Comment by ravindk89** [comment]
RFM, hold merge until release.

**Review by kannappanr — APPROVED** [review]
LGTM

**Review by harshavardhana — APPROVED** [review]

File: source/reference/minio-server/minio-server.rst [hunk 1]
```diff
@@ -253,6 +263,45 @@ Root Credentials
...
+   Specify ``off`` to disable the embedded MinIO Console.
+
+.. envvar:: MINIO_BROWSER_REDIRECT
+
+   *Optional*
+
+   Specify ``false`` to direct MinIO to prevent redirecting browsers accessing
+   the ``localhost:9000`` URL to the MinIO Console. MinIO instead throws an
+   XML access error.
+
```
Comment (harshavardhana) [inline source/reference/minio-server/minio-server.rst:304]: This env is removed @ravindk89 let's remove this before publishing.

Comment (ravindk89) [reply source/reference/minio-server/minio-server.rst:304]: Whoops - missed this comment. Fixing.

**Pull request merged**